packages/generator/src/v2.0/generatePerActionDataFiles.ts

```diff
@@ -188,16 +172,43 @@ export async function generatePerActionDataFiles(
   
           }
 
+          
           ${jsDocBlock.join("\n")}
           */
-          export type ${action.shortApiName} = ${action.shortApiName}.Signatures;
+          export interface ${action.shortApiName} extends ActionDefinition<"${action.shortApiName}", ${uniqueApiNamesString}, ${action.shortApiName}.Signatures> {
```

Comment:
Just double checking that hovering over `client(myAction)` will give the right docs?

Reply:
It does.